Classes/MOC/NotFound/ViewHelpers/RequestViewHelper.php

```diff
@@ -134,7 +134,7 @@ protected function appendFirstUriPartIfValidDimension(&$path) {
 				}
 			}
 		} else {
-			if (count($firstUriPartExploded) === count($dimensionPresets)) {
+			if (count($firstUriPartExploded) !== count($dimensionPresets)) {
```

Comment:
seems to correspond with https://github.com/neos/neos-development-collection/blob/4a1495e0e5d6eff49bb037b380ca03f5544ddc66/TYPO3.Neos/Classes/TYPO3/Neos/Routing/FrontendNodeRoutePartHandler.php#L538